examples/features/metadynamics/README.md

```diff
@@ -15,3 +15,5 @@ i-pi--driver -u -h zundel -m zundel
 ```
 
 However, they require having PLUMED libraries and Python bindings in the appropriate system paths.
+
+Note also in `pimd_metadynamics` how to retrieve CVs computed PLUMED-side back into i-PI.
```

Comment:
Can you be a bit more verbose here that the CVs or CV attributes can be specified in <plumedextras> </plumedextras> and printed out as a trajectory by defining the `extras_type` attribute of `extras bias`.

Reply:
Can you also explain to me if we need to specify the variables in forces_extras that need to be printed?

Reply:
So, if I understand correctly,  if one does stuff classically, then the variables need not be defined in forces_extras. However, for PI simulations, the user will need to as trajectories are only printed for the beads?

Reply:
Yes. `forces_extras` is needed to tell i-PI that the named extras (1) have a numerical value and (2) can be Fourier interpolated/extrapolated to compute them for the full ring polymer. The docs for forces_extras says it well
```
A list of quantities that should be extracted from the ‘extra’ string returned by the forcefield, that are then treated the same way as energy or forces – that is treated as a numerical, physical quantity, interpolated when changing the number of PI replicas. Same quantities from different force components are summed as well. The names should correspond to entries in the JSON-formatted extra string.
```
Maybe we can use a more telling name as `extras_interpolation` ?

Reply:
yeah, or how about 'ring_polymer_interpolation_extras' or 'ring_polymer_extras'?

Reply:
ok, I have now pushed this and I also ended up adding `forces_component` output. In analogy with `pot_component`, by default it'll contract to the centroid so this should cover your use case. PLS test that it works before we merge - in all events I've given up on having a finalized ensemble-demo before submission.